File: Software Technologies/Exam - 21 Dec 2017/CSharp/LogNoziroh/Models/Report.cs
using System.ComponentModel.DataAnnotations;

namespace LogNoziroh.Models
{
    public class Report
    {
        public int Id { get; set; }

        [Required]
        public string Status { get; set; }

        [Required]
        public string Message { get; set; }

        [Required]
        public string Origin { get; set; }


    }
}

File: Software Technologies/Exam - 21 Dec 2017/JS/models/Report.js
const mongoose = require('mongoose');

let reportSchema = mongoose.Schema({
    status: {type: 'string', required: 'true'},
    origin: {type: 'string', required: 'true'},
    message: {type: 'string', required: 'true'},
});

let Report = mongoose.model('Report', reportSchema);

module.exports = Report;